granulate_utils/linux/cgroups/base_cgroup.py

```diff
@@ -5,17 +5,61 @@
 
 import os
 from pathlib import Path
-from typing import List, Tuple
+from typing import List, Mapping, Optional, Tuple
 
 from psutil import NoSuchProcess
 
-CGROUPFS = Path("/sys/fs/cgroup")
-SUBSYSTEMS = {"memory", "cpu"}
-
+from granulate_utils.exceptions import AlreadyInCgroup
+from granulate_utils.linux.mountinfo import iter_mountinfo
+from granulate_utils.linux.ns import resolve_host_root_links
 
-class AlreadyInCgroup(Exception):
```

Comment:
this is exported in __init__ - please change the path there

Reply:
Done